userguide/tutorials/aws-container.adoc

```diff
@@ -59,15 +69,17 @@ image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-
 [start=6]
 . Choose FARGATE as the launch type and select an existing VPC and subnet:
 
-image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-run-task-config.png[align=center]
+image:ecs-run-task.png[align=center]
 
```

Comment:
Should'nt the image name be `ecs-run-task-config.png`?

Reply:
@reshmabidikar - Fixed...please check